src/components/SingleArticle.jsx

```diff
@@ -20,6 +21,8 @@ const SingleArticle = () => {
             <img className="single-article-img" src={article.article_img_url} />
             <p> Votes: {article.votes} </p>
```

Comment:
see the following resource on labelling: https://www.refactoringui.com/previews/labels-are-a-last-resort